gulpfile.js

```diff
@@ -29,7 +30,17 @@ elixir(function(mix) {
     		'public/css/app.css'
     	], {
     		liveCSS: true
-    	});
+    	})
+        .karma({
+            jsDir:[
+                'public/js/vendor.js',
+                'node_modules/angular-mocks/angular-mocks.js',
+                'public/js/partials.js',
+                'public/js/app.js',
+                'tests/angular/**/*spec.js'
```

Comment:
Oh and I just remembered something
Another cool way of doing this, is by adding a `.spec.js` file in the same folder
so if you're testing a certain service, you can have the test file right next to it.
just like what they do in this [awesome generator](https://github.com/Swiip/generator-gulp-angular)
Let me know what you think